File: node/trello/card/read.js
var Read = {
  init: function(configFileName)
  {
    require('../base/read').init(Read, configFileName);
    Read.baseUrl = Read.config.getApi('card', 'item');
  },

  info: function(id)
  {
    Read.trello.getData(
      Read.baseUrl + '/' + id,
      '&fields=name,desc,due,dateLastActivity,closed,subscribed,url,labels',
      function(info) {
        if (info) {
          console.log('名称：%s（%s）', info.name, info.id);
          console.log('关闭：%s', info.closed ? '是' : '否');
          console.log('订阅：%s', info.subscribed ? '是' : '否');
          if(info.labels) {
            var labels = [];
            for (var index in info.labels) {
              labels.push(info.labels[index].name);
            }
            console.log('标签：%s', labels.join('，'));
          }
          console.log('链接：%s', info.url);
          console.log('过期时间：%s', info.due);
          console.log('最后活动时间：%s', info.dateLastActivity);
          console.log('描述：%s', info.desc);
        }
      }
    );
  }
};

module.exports = Read;

File: python/trello/card/read.py
#-*- coding:utf-8 -*-
from trello.base.read import Read as ReadBase

class Read(ReadBase):
  def __init__(self, configFileName):
    super(Read, self).__init__(configFileName)
    self.baseUrl = self.config.getApi('card', 'item')

  def info(self, id):
    info = self.trello.getData(
      '%s/%s' % (self.baseUrl, id),
      '&fields=name,desc,due,dateLastActivity,closed,subscribed,url,labels'
    )
    if info:
      print('名称：%s（%s）' % (info['name'], info['id']))
      print('关闭：%s' % ('是' if info['closed'] else '否'))
      print('订阅：%s' % ('是' if info['subscribed'] else '否'))
      if info['labels']:
        labels = [label['name'] for label in info['labels']]
        print('标签：%s' % '，'.join(labels))
      print('链接：%s' % info['url'])
      print('过期时间：%s' % info['due'])
      print('最后活动时间：%s' % info['dateLastActivity'])
      print('描述：%s' % info['desc'])

  def members(self, id):
    super(Read, self).members(
      '%s/%s/members' % (self.baseUrl, id)
    )
